djangoevents/__init__.py

```diff
@@ -19,9 +21,20 @@
     'entity_mutator',
     'singledispatch',
     'publish',
+    'store_event'
     'subscribe',
     'unsubscribe',
     'subscribe_to',
     'BaseEntity',
     'EventSourcingWithDjango'
 ]
+
+
+def publish(event):
+    warnings.warn("`publish` is depreciated. Please switch to: `store_event`", DeprecationWarning)
+    return es_publish(event)
+
+
+def store_event(event):
```

Comment:
why we use "store_event" instead of "publish"? It is called "publish" because the "publish" and "store" are decoupled. "publish" publishes the event to an event bus and "store" is a listener on the event bus and save all events into the event store. As a result, we have the ability to attach custom event handler to the event bus and handle published events. by renaming this to "store_event", we lose the understanding of the flexibility of the library. @lukaszb @piotrekno1